app/components/impagination-dataset.js

```diff
@@ -1 +1,116 @@
-export { default } from 'ember-impagination/components/impagination-dataset';
```

Comment:
Typically the way that a component is defined in an addon is in `addon/components` and then exported as-is from `app/components`. Unless there is a compelling reason, we should probably follow this pattern.